### PR TITLE
Ravinder | Test | Fixed daybook table issue

### PR DESCRIPTION
**Pull request by ravinderwalkover**
https://app.clickup.com/t/7hhh87

### CODE REVIEW

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 1]
```diff
@@ -98,6 +101,24 @@ $custom-typography: mat-typography-config(
...
+            }
+            tr:last-child {
+                border-bottom: 0px !important;
+                margin-bottom: 0px;
+            }
+        }
+        &.hide-td {
+            display: none !important;
+        }
+        .justify-content-end {
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/assets/css/style-2.scss:119]: please edit this class to justify-content-start if we are using for flex-start.

File: apps/web-giddh/webpack.test.partial.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = {
             'errlyticsNeeded': JSON.stringify(false),
             'errlyticsKey': JSON.stringify(''),
             'AppUrl': JSON.stringify('https://test.giddh.com/'),
-            'ApiUrl': JSON.stringify('https://apitest.giddh.com/'),
+            'ApiUrl': JSON.stringify('https://apirelease.giddh.com/'),
```
Comment (aditi-tiwari17) [inline apps/web-giddh/webpack.test.partial.js:14]: it should be reverted i guess?

File: apps/web-giddh/src/app/daybook/daybook.component.html [hunk 3]
```diff
@@ -498,39 +475,62 @@
...
+                                                [attr.data-title]="commonLocaleData?.app_warehouse"
+                                            >
                                                 {{ details?.inventory?.warehouse?.name }}
                                             </td>
                                         </ng-container>
-                                        <tr mat-header-row [hidden]="true" *matHeaderRowDef="tableExpandedColumns"></tr>
-                                        <tr mat-row *matRowDef="let entry; columns: tableExpandedColumns"></tr>
+                                        <tr
+                                            mat-header-row
+                                            [hidden]="true"
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/daybook/daybook.component.html:511]: Please check hidden is set to true

File: apps/web-giddh/webpack.test.partial.js (hunk 2 above)
Comment (ravinderwalkover) [reply apps/web-giddh/webpack.test.partial.js:14]: we changed because apitest is not working and test is on hold

**Review by prateek-giddh — APPROVED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

**Pull request merged**